hw3/01-characters.js

```diff
@@ -1,4 +1,43 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
 
-fetch(url);
+fetch(url)
```

Comment:
great job on exercise one I like how your code is compact and it works great. I only have one recommendation which is that the wave audit has errors related to missing alt text for the images. that's the only thing I would recommend for this one.